gamemodes/cinema/gamemode/theater/sh_protect.lua

```diff
@@ -77,7 +77,45 @@ if CLIENT then
         weapon_physgun = true,
         weapon_slitter = true,
         weapon_gauntlet = true,
-        weapon_peacekeeper = true
+        weapon_peacekeeper = true,
+        weapon_magicmissile = true,
+        weapon_garand = true,
+        weapon_pickaxe = true,
+        weapon_pickaxe_diamond = true,
+        weapon_fists = true,
+        weapon_crusadersword = true,
+        gun_44magnum = true,
```

Comment:
Just do class:StartWith("gun_")

Reply:
I didn't see this in the API or was even aware of string.StartWith exists. Will update accordingly.